conf/viewvc.conf.dist

```diff
@@ -114,8 +114,9 @@
 ## root_parents: Specifies a list of directories under which any
 ## number of repositories may reside.  You can specify multiple root
 ## parents separated by commas or new lines, each of which is of the
-## form "path: type" (where the type is either "cvs" or "svn", and
-## the path is an absolute filesystem path).
+## form "path [= context] : type" (where the type is either "cvs" or
+## "svn", the path is an absolute filesystem path, and the (optional)
+## context is prepended to each repository's name).
 ##
```

Comment:
I think this concept of "context" in root names might require a bit more documentation, perhaps up in the `SOME TERMINOLOGY USED HEREIN` section of the file's header.  Also, you don't really mention the alternative syntax in the `cvs_roots`, `svn_roots`, or `renamed_roots` configuration docs/examples themselves, though the Pull Request description seems to indicate that those are likewise affected by the changes (unless I'm just misunderstanding).